<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,0">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::FullViewportUpdate</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,0,0,1,0,0,0,0">
        <item>
         <widget class="QCommandLinkButton" name="btn_newnode">
          <property name="text">
           <string>Добавить узел</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>от:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_from"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>до:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_to"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>длина:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_length"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chb_isFake">
          <property name="text">
           <string>Фиктивная</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btn_addrelation">
          <property name="text">
           <string>Добавить связь</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCommandLinkButton" name="btn_delete">
            <property name="text">
             <string>Удалить узел</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_delete"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btn_findCritical">
          <property name="text">
           <string>Найти критический путь</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_minlength"/>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btn_makeTable">
          <property name="text">
           <string>Построить таблицу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btn_makePlot">
          <property name="text">
           <string>Построить календарный график</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="readmenu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="readmenu">
   <property name="text">
    <string>Чтение из таблицы...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
